### PR TITLE
Fixes VSTS Bug 753551: Using statement indenting

### PR DESCRIPTION
**Pull request by mkrueger**
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/753551

### CODE REVIEW

**Comment by mkrueger** [comment]
@monojenkins backport release-7.8

File: main/src/addins/CSharpBinding/MonoDevelop.CSharp.Formatting/CSharpIndentationTracker.cs [hunk 1]
```diff
@@ -69,7 +72,22 @@ public override string GetIndentationString (int lineNumber)
 			if (indentation.HasValue && indentation.Value > 0) {
 				return CalculateIndentationString (indentation.Value);
 			}
-			return editor.GetLineIndent (lineNumber) + CalculateIndentationString (editor.Options.IndentationSize);
+
+			var line = editor.GetLine (lineNumber);
+			if (line == null)
+				return editor.GetLineIndent (lineNumber);
+			try {
+				var syntaxRoot = doc.GetSyntaxRootSynchronously (default);
```
Comment (Therzok) [inline main/src/addins/CSharpBinding/MonoDevelop.CSharp.Formatting/CSharpIndentationTracker.cs]: move this inside the if statement.

**Comment by mkrueger** [comment]
@monojenkins backport release-7.8

**Comment by slluis** [comment]
@mkrueger build is failing. Can you update to latest master?

**Pull request merged**